osbs/api.py

```diff
@@ -418,7 +418,7 @@ def create_source_container_pipeline_run(self,
 
     @osbsapi
     def get_build_name(self, build_response):
-        return build_response['metadata']['name']
+        return build_response.pipeline_run_name
```

Comment:
Could we have a type annotation for the build_response?

Reply:
added